code/rdb/datastripe.q

```diff
@@ -0,0 +1,3 @@
+\d .ds
+
+segmentid: "J"$.proc.params[`segid]		// segmentid variable defined by applying key to dictionary of input values
```

Comment:
This is out of date I'm sure, this file should no longer exist, I don't think. Can you make sure your branch is up to date?